server/guiserver/apps.py

```diff
@@ -106,6 +106,7 @@ def server():
         'jujugui.uuid': options.uuid,
         'jujugui.interactive_login': options.interactivelogin,
         'jujugui.gzip': options.gzip,
+        'jujugui.GTM_enabled': options.gtm,
```

Comment:
So...I can get you partway there. I can do gtm_enabled everywhere except the jujugui.GTM_enabled bit, and in the tornado start script where underscores are an issue. That can be your pick of 'gtm' or 'gtmenabled'.